crates/sargon/src/types/invalid_transaction_if_neglected.rs

```diff
@@ -4,24 +4,24 @@ use crate::prelude::*;
 /// neglect certain factor source, either by user explicitly skipping
 /// it or if implicitly neglected due to failure.
 #[derive(Clone, Debug, PartialEq, Eq, std::hash::Hash)]
-pub struct InvalidTransactionIfNeglected {
+pub struct InvalidTransactionIfNeglected<S: Signable> {
```

Comment:
the whole `Signable` trait is not used here right? So this might be generic over `T: SignableID` , right? which I think is a good idea, easier to test this in isolation, right?